src/common/Gateway.sol

```diff
@@ -233,8 +233,8 @@ contract Gateway is Auth, IGateway, IRecoverable {
         bytes32 messageHash = keccak256(message);
         uint256 recovery = recoveries[chainId][adapter][messageHash];
 
-        require(recovery != 0, "Gateway/message-recovery-not-initiated");
-        require(recovery <= block.timestamp, "Gateway/challenge-period-has-not-ended");
+        require(recovery != 0, MessageRecoveryNotInitiated());
+        require(recovery <= block.timestamp, MessageRecoveryPeriodNotEnded());
```

Comment:
I feel the challenge period terminology here is helpful. Since now it seems like there is a period in which the recovery is handled.

Reply:
Will add! 🙏🏻